waspc/run

```diff
@@ -96,7 +97,7 @@ ORMOLU_CHECK_CMD="$ORMOLU_BASE_CMD --mode check "'$'"(git ls-files '*.hs' '*.hs-
 ORMOLU_FORMAT_CMD="$ORMOLU_BASE_CMD --mode inplace "'$'"(git ls-files '*.hs' '*.hs-boot')"
 
 echo_and_eval() {
-  echo -e $"${LIGHT_CYAN}Running:${DEFAULT_COLOR}" "$1" "\n"
+  echo -e $"${LIGHT_CYAN}Running:${DEFAULT_COLOR}" "$1" "\n" 1>&2
```

Comment:
Conventionally, human-readable info messages get written to stderr and not stdout.

Otherwise, when we do `my_var=$(./run get-version)`, the contents of `$my_var` would be `Running: blah\n0.19.0` insted of `0.19.0`